src/pages/Dashboard/Markets/SupplyMarket/CollateralConfirmModal/index.tsx

```diff
@@ -16,18 +17,23 @@ export const CollateralConfirmModal: React.FC<IConfirmCollateralModalProps> = ({
   handleClose,
 }) => {
   const styles = useStyles();
+  const { t } = useTranslation();
   return (
     <Modal
       className="venus-modal"
       isOpened={!!asset}
       handleClose={handleClose}
-      title={`${asset?.collateral ? 'Disable' : 'Enable'} as collateral`}
+      title={`${
+        asset?.collateral
+          ? t('markets.collateralConfirmModal.disable')
+          : t('markets.collateralConfirmModal.enable')
+      } ${t('markets.collateralConfirmModal.asCollateral')}`}
```

Comment:
Once these are translated, the order of the words might not be the same depending on the language it is translated to. For that reason, we should include the full sentence for each string:
```typescript
title={asset?.collateral ? t('markets.collateralConfirmModal.disableAsCollateral') : t('markets.collateralConfirmModal.enableAsCollateral') }
```